underpants.js

```diff
@@ -80,23 +113,52 @@ var _ = {};
 *   _.last(["a", "b", "c"], 2) -> ["b", "c"]
 */
 
+// last should accept an array and a number
+// if arr is not an array or num is negative, return []
+// if num is not a number, return last element
+// if num > arr.length, return whole array
+// return last {number} amount of items in the array otherwise
+
+_.last = (arr, num) => {
+    if (!(Array.isArray(arr)) || num < 0 || isNaN(num)) {
+        return isNaN(num) ? arr[arr.length - 1] : [];
+    }
+    if (num > arr.length) {
+        return arr;
+    }
+    return arr.slice(arr.length - num, arr.length);
```

Comment:
```suggestion
    return arr.slice(arr.length - num);
```

You should be able to leave this second arg off. According to [the MDN docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice), if the second argument is omitted, slice will go through the end of the array.